modules/Behaviour/moduleFunctions.php

```diff
@@ -58,8 +58,12 @@ function getBehaviourRecord(ContainerInterface $container, $gibbonPersonID)
                 ->sortBy('timestamp', 'DESC')
                 ->fromPOST($schoolYear['gibbonSchoolYearID']);
 
-            $behaviourRecords = $behaviourGateway->queryBehaviourRecordsByPerson($criteria, $schoolYear['gibbonSchoolYearID'], $gibbonPersonID);
-
+            if (empty($gibbonPersonIDCreator)) {              
+                $behaviourRecords = $behaviourGateway->queryBehaviourRecordsByPerson($criteria, $schoolYear['gibbonSchoolYearID'], $gibbonPersonID);
+            } else {
+                $behaviourRecords = $behaviourGateway->queryBehaviourRecordsByPerson($criteria, $schoolYear['gibbonSchoolYearID'], $gibbonPersonID, $gibbonPersonIDCreator);
+            }
```

Comment:
It looks like this conditional can be simplified even further, since if $gibbonPersonIDCreator is already null, it will be passed as a null value into the method, which does its own empty() check internally.
```suggestion  
            $behaviourRecords = $behaviourGateway->queryBehaviourRecordsByPerson($criteria, $schoolYear['gibbonSchoolYearID'], $gibbonPersonID, $gibbonPersonIDCreator);
```